pipeline-runner/init.R

```diff
@@ -14,14 +14,50 @@ options(keep.source.pkgs = TRUE)
 for (f in list.files('R', '.R$', full.names = TRUE)) source(f, keep.source = TRUE)
 load('R/sysdata.rda') # constants
 
+buildActivityArn <- function(aws_region, aws_account_id, activity_id) {
+    if(is.na(activity_id)) {
+        return(NA)
+    }
+
+    activity_arn <- sprintf("arn:aws:states:%s:%s:activity:%s", aws_region, aws_account_id, activity_id)
+    message("Built activity ARN: ", activity_arn)
+    return(activity_arn)
+}
+
 load_config <- function(development_aws_server) {
+    label_path <- "/etc/podinfo/labels"
+    aws_account_id <- Sys.getenv("AWS_ACCOUNT_ID", unset="242905224710")
```

Comment:
What is the `unset` default value?

Reply:
unset is just the parameter name, the value is the aws account id used throughout the platform

Reply:
it means if `AWS_ACCOUNT_ID` is not set (unset) use that default value